sky/jobs/utils.py

```diff
@@ -972,6 +972,9 @@ def dump_managed_job_queue() -> str:
         job['status'] = job['status'].value
         job['schedule_state'] = job['schedule_state'].value
 
+        if not job['metadata']:
+            job['metadata'] = {}
+
```

Comment:
Why is this needed? Default should be `{}`, right?

Reply:
just in case its a `None` for some reason

Reply:
Looking at get_managed_jobs source, that doesn't seem possible. I'd rather an assert if it's necessary.